libs/shared/advert-card/src/lib/shared-advert-card.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { number, text } from '@storybook/addon-knobs';
```

Comment:
+ object

Reply:
Unsure about this, could you expand on this please